### ISSUE TITLE
DiagTrack error - serious or not? App said to report issue

### ISSUE THREAD

**Issue by lumodon** [10se1ucgo/DisableWinTracking#406]
**INFORMATION:**

* Windows Edition (Home/Pro/Enterprise):
* Architecture (32bit/64bit):
* Options used:

**LOG:**

```python
12:33:45 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
12:33:45 INFO: ('Windows', 'DESKTOP-NALJ59E', '10', '10.0.16299', 'AMD64', 'Intel64 Family 6 Model 142 Stepping 10, GenuineIntel')
12:33:45 INFO: DisableWinTracking version 3.2.1
12:35:02 INFO: IP Blocker: The IP 134.170.30.202 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 137.116.81.24 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 157.56.106.189 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 2.22.61.43 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 2.22.61.66 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 204.79.197.200 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 23.218.212.69 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 64.4.54.254 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 65.39.117.230 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 65.52.108.33 was successfully blocked.
12:35:02 INFO: IP Blocker: The IP 65.55.108.23 was successfully blocked.
12:35:02 INFO: DiagTrack: Successfully deleted service 'DiagTrack'
12:35:02 INFO: DiagTrack: Completed Part 1/3
12:35:02 INFO: DiagTrack: Successfully deleted service 'dmwappushservice'
12:35:02 INFO: DiagTrack: Completed Part 2/3
12:35:02 INFO: DiagTrack: Successfully erased tracking log.
12:35:02 INFO: DiagTrack: Completed Part 3/3
12:35:02 INFO: DiagTrack: Completed successfully, without errors.
12:35:02 ERROR: Services: Failed to remove service 'DiagTrack'
Traceback (most recent call last):
  File "dwt_util.py", line 168, in delete_service
  File "site-packages\win32\lib\win32serviceutil.py", line 303, in RemoveService
error: (1072, 'DeleteService', 'The specified service has been marked for deletion.')
12:35:03 INFO: Registry: Successfully modified dmwappushsvc key.
12:35:03 INFO: Registry: Successfully modified DiagTrack key.
12:35:03 INFO: Registry: Successfully modified AllowTelemetry key.
12:35:03 INFO: Registry: Successfully modified ListPin key.
12:35:03 INFO: Registry: Successfully modified ListPin64Bit key.
12:35:03 INFO: Registry: Successfully modified FileSync key.
12:35:04 INFO: OneDrive: successfully uninstalled
12:35:04 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
12:35:04 INFO: If you feel something didn't work properly, please press the 'Report an issue' button and follow the directions

```

**Comment by Ruined1** [comment]
Not serious, the service was already marked for deletion. I really gotta add a clause for this, because no one ever search for duplicate bugs and this has been reported at least 20 times.

**Comment by Ruined1** [comment]
There is already code to handle this:
```
if output[0] == 1072:
	logger.info("DiagTrack: {0} service marked for deletion. This is OK, make sure you reboot your machine!".format(service[1]))
```
Need to find out why this isn't working.

**Comment by Ruined1** [comment]
Fixed in version 3.2.2
```
19:39:52 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)] on win32
19:39:52 INFO: ('Windows', 'User', '10', '10.0.17763', 'AMD64', 'AMD64 Family 23 Model 8 Stepping 2, AuthenticAMD')
19:39:52 INFO: DisableWinTracking version 3.2.2
19:40:20 INFO: DiagTrack: Successfully deleted service 'DiagTrack'
19:40:20 INFO: DiagTrack: Completed Part 1/3
19:40:20 INFO: DiagTrack: Successfully deleted service 'dmwappushservice'
19:40:20 INFO: DiagTrack: Completed Part 2/3
19:40:20 INFO: DiagTrack: Successfully erased tracking log.
19:40:20 INFO: DiagTrack: Completed Part 3/3
19:40:20 INFO: DiagTrack: Completed successfully, without errors.
19:40:20 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
19:40:20 INFO: If you feel something didn't work properly, please press the 'Report an issue' button and follow the directions
```
